tools/devices/lpc/lpc111.xml

```diff
@@ -1,103 +0,0 @@
-      <gpio port="0" pin="10" func="Swd,Gpio,Spi0,Timer16_0"     bits="mode,hys,od" define="SWCLK_PIO0_10" />
-      <gpio port="0" pin="11" func="Reserved,Gpio,Adc,Timer32_0" bits="mode,hys,admode,od"/>
-      <gpio port="1" pin="0"  func="Reserved,Gpio,Adc,Timer32_1" bits="mode,hys,admode,od"/>
-      <gpio port="1" pin="1"  func="Reserved,Gpio,Adc,Timer32_1" bits="mode,hys,admode,od"/>
-      <gpio port="1" pin="2"  func="Reserved,Gpio,Adc,Timer32_1" bits="mode,hys,admode,od"/>
-      <gpio port="1" pin="3"  func="Swd,Gpio,Adc,Timer32_1"      bits="mode,hys,admode,od" define="SWDIO_PIO1_3" />
-      <gpio port="1" pin="4"  func="Gpio,Adc,Timer32_1"          bits="mode,hys,admode,od"/>
-      <gpio port="1" pin="5"  func="Gpio,Uart,Timer32_0"         bits="mode,hys,od"/>
-      <gpio port="1" pin="6"  func="Gpio,Uart,Timer32_0"         bits="mode,hys,od"/>
-      <gpio port="1" pin="7"  func="Gpio,Uart,Timer32_0"         bits="mode,hys,od"/>
-      <gpio port="1" pin="8"  func="Gpio,Timer16_1"              bits="mode,hys,od"/>
-      <gpio port="1" pin="9"  func="Gpio,Spi1,Timer16_1"         bits="mode,hys,od"/>
-      <gpio port="1" pin="10" func="Gpio,Spi1,Adc,Timer16_1"     bits="mode,hys,admode,od"/>
-      <gpio port="1" pin="11" func="Gpio,Adc,Timer16_1"          bits="mode,hys,admode,od"/>
-      <gpio port="2" pin="0"  func="Gpio,Uart,Spi1"              bits="mode,hys,od"/>
-      <gpio port="2" pin="1"  func="Gpio,Uart,Spi1"              bits="mode,hys,od"/>
-      <gpio port="2" pin="2"  func="Gpio,Uart,Spi1"              bits="mode,hys,od"/>
-      <gpio port="2" pin="3"  func="Gpio,Uart,Spi1"              bits="mode,hys,od"/>
-      <gpio port="2" pin="4"  func="Gpio,Spi1,Timer16_1"         bits="mode,hys,od"/>
-      <gpio port="2" pin="5"  func="Gpio,Timer32_0"              bits="mode,hys,od"/>
-      <gpio port="2" pin="6"  func="Gpio,Timer32_0"              bits="mode,hys,od"/>
-      <gpio port="2" pin="7"  func="Gpio,Uart,Timer32_0"         bits="mode,hys,od"/>
-      <gpio port="2" pin="8"  func="Gpio,Uart,Timer32_0"         bits="mode,hys,od"/>
-      <gpio port="2" pin="9"  func="Gpio,Timer32_0"              bits="mode,hys,od"/>
-      <gpio port="2" pin="10" func="Gpio"                        bits="mode,hys,od"/>
-      <gpio port="2" pin="11" func="Gpio,Spi1,Timer32_0"         bits="mode,hys,od"/>
-      <gpio port="3" pin="0"  func="Gpio,Uart,Timer16_0"         bits="mode,hys,od"/>
-      <gpio port="3" pin="1"  func="Gpio,Uart,Timer16_0"         bits="mode,hys,od"/>
-      <gpio port="3" pin="2"  func="Gpio,Uart,Spi1,Timer16_0"    bits="mode,hys,od"/>
-      <gpio port="3" pin="3"  func="Gpio,Uart,Timer16_0"         bits="mode,hys,od"/>
-      <gpio port="3" pin="4"  func="Gpio,Uart,Timer16_0"         bits="mode,hys,od"/>
-      <gpio port="3" pin="5"  func="Gpio,Uart,Timer16_0"         bits="mode,hys,od"/>
-    </driver>
-  </device>
-</modm>
```

Comment:
Why were these moved to modm-devices? They are not at the level of quality or automation that warrants them being there.